pymc3/sampling.py

```diff
@@ -353,7 +352,7 @@ def sample(
     Notes
     -----
     Optional keyword arguments can be passed to ``sample`` to be delivered to the
-    ``step_method``s used during sampling.
+    ``step_method``\ s used during sampling.
```

Comment:
On `master`:

![image](https://user-images.githubusercontent.com/33491632/100882268-e79d6700-34a6-11eb-8270-e352468b84d1.png)


This branch:

![image](https://user-images.githubusercontent.com/33491632/100882209-d5232d80-34a6-11eb-871e-9345e5a08f30.png)